Assets/Scripts/System/GameSync.cs

```diff
@@ -76,7 +76,9 @@ public void execute()
 
     private void fetchPlaylistSubscriptions()
     {
-        WWW www = new WWW(library_url + "/api/v1/playlists/?api_key=" + api_key);
+        Dictionary<string, string> headers = new Dictionary<string, string>();
+        headers.Add("User-Agent", "Winnitron Launcher/2.0.0-rc2 (http://winnitron.com)");
```

Comment:
I know I'm hard-coding this here, but I'm not sure how to read the `versionNumber` property on `GM`. My Unity skills :sob: